changelog.d/8004.feature

```diff
@@ -0,0 +1 @@
+Insert a confirmation page between clicking the email link and resetting your password to prevent mishaps.
```

Comment:
Require the user to confirm that their password should be reset after clicking the email confirmation link.